### ISSUE TITLE
오후 돌발이벤트 기능구현 계획

### ISSUE THREAD

**Issue by GTU9** [Java-TeamProject/MISA#11]
##작성자
김상익

##맡은 파트
오후 돌발이벤트

##구현할 기능
오후 고정이벤트에서 확률적으로 연계되어 나타나는 돌발이벤트의 기능 구현

##선택지 문서화 
==================================================================
1.고정 대학사람들과 재밌는 술자리 선택 이벤트 연계 선배와의 술자리 돌발 이벤트
-> 친구들과 술을 마시고 집을 가는 도중 졸업 후 취업한 친한 선배를 만났다. 그 친한 선배가 나에게 끝나고 나랑 
        따로 술을 더 마시자고 한다. 당신의 선택은?
        ->>친한 선배와 술을 좀 더 같이 마신다. {보통 -체력, 보통 +인망}
        ->>거절하고 그대로 집으로 돌아가자 {변동 수치 없음}

2.유튜브로 통한 휴식 시간 선택 이벤트 연계 게임 하기 돌발 이벤트
-> 유튜브를 보면서 휴식중에 내가 하는 게임의 프로게이머의 슈퍼플레이 영상을 보았다.
   영상을 보니 나도 이렇게 할 수 있을 거 같다는 자신감이 생긴다. 게임을 조금만 해볼까?
        ->>내 안에 게이머의 혼이 불탄다. 이 정도는 나도 할 수 있을 것 같다. 게임하러가자! {50프로 확률로 게임을 패배, 승리한다.       
               패배시 보통 –체력, 승리시 많은  체력}  
        ->>내 주제에 이렇게 어떻게 하냐 괜히 게임했다가 기분만 상하지 말고 유튜브나 계속 보자{변동 수치 없음}

3. 일찍 잠드는 선택 이벤트 연계 옆집의 죽을 것같은 목소리 돌발 이벤트
-> 잠을 자고 있는데 평소에 잘 알고 지내던 옆집에서 죽을 듯한 목소리가 들린다. 119를 부르고 응급실 
     에 데려가야 할까?
        ->>잠을 뒤로 하고, 응급실에 데려간다. {적은 -체력, 보통 +인망}
        ->>잠꼬대하는 거겠지 잠이나 자자. {변동 수치 없음}

**Comment by GTU9** [comment]
오후 돌발이벤트 수치 변화까지 완성했습니다.

**Comment by ohdyo** [comment]
네 확인했습니다. 수치 조정은 다 통합하고 14일 돌려봐서 평균이 어느정도 나오는지 확인하고 정산하면 끝날것 같네요

**Comment by ddoddo1228** [comment]
수치 조정만 하면 될 것 같네요 수고하셨습니다~

**Comment by Baeminseo** [comment]
구체적으로 잘 짜셨고 저랑 무사히 충돌없이 합치면 되겠네요! 수고하셨습니다~!
